### ISSUE TITLE
failed to run node on mac os x 10.13.1

### ISSUE THREAD

**Issue by eliwe** [lamdu/lamdu#90]
Installed as described in the doc.
Lamdu works but fails to run node with the following printed:
`lamdu/.stack-work/install/x86_64-osx/lts-10.5/8.2.2/share/x86_64-osx-ghc-8.2.2/nodejs-exec-0.1.0.0/bin/node: bad option: --harmony-tailcalls`

I have installed node 6.13.0 using brew. but lamdu/.stack-work/install/x86_64-osx/lts-10.5/8.2.2/share/x86_64-osx-ghc-8.2.2/nodejs-exec-0.1.0.0/bin/node is 9.5.0. I don't know where it comes from. Node 9.5 doesn't have the --harmony-tailcalls option.

**Comment by ysangkok** [comment]
Tail calls seem not to be supported in newer Node versions according to http://node.green/ ... I guess the version check in [build_node](https://github.com/lamdu/nodejs-exec/blob/master/build_node.sh) could also build node if the installed version is too new, but this is not a viable strategy...

**Comment by ysangkok** [comment]
Seems like standardization is not happening: https://github.com/tc39/proposal-ptc-syntax/issues/22

**Comment by eliwe** [comment]
Before installing node v6.13.0 i had node v9.5 installed through brew.
Then i ran stack install in the lambdu dir. Tried to run it an understood that it will not work with node v9.
I uninstalled node v9 and installed v6 but stack had cached the v9 in its dir and refused to update  the node executable. I did git clean -fdx and ran install again.
This solved the issue. 
But any way the documentation should say that node v6 is needed.
